docs/products/valkey.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Aiven for Valkey
+---
+
+Aiven for Valkey is a managed, in-memory NoSQL database service that offers high performance, scalability, and security.
+
+Developed under the Linux Foundation, Valkey is an open-source fork of Redis® designed
+to provide a seamless and reliable alternative to Redis OSS.  Aiven for Valkey
+ensures full compatibility with Redis OSS 7.2.4 and Aiven for Caching, making it easy
+for users to transition their existing applications without disruption.
+
+:::tip
+Since Aiven for Valkey is the new open-source fork of Redis®, and Aiven for Caching
```

Comment:
shall we add a sentence saying that Valkey 7.2.5 remains to be full comaptibility with Aiven for Caching, hence, customer can refer to the old docs etc.